scripts/generate-google3-docs/resource-reference/generated/resource-docs/cloudquota/apiquotaadjustersettings.md

```diff
@@ -28,7 +28,7 @@
 </tr>
 <tr>
 <td>{{product_name_short}} Resource Short Names</td>
-<td>APIQuotaAdjusterSettings<br>gcpapiquotaadjustersettings<br>apiquotaadjustersettings<br>apiquotaadjustersettings</td>
```

Comment:
Was this change driven by a change in template? Or was this manual?

Reply:
Related to https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/4815

In short, some kinds have special plural/singular format but the KCC reference doc generator handles it incorrectly, fixed in 4815.